src/components/MovieAdder.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { useState, ChangeEvent } from "react";
+import AsyncSelect from "react-select/lib/Async";
+import { debounce } from "underscore";
+
+import { MoviePreference, User } from "../types";
+
+const promiseOptions = (inputValue: string) => {
```

Comment:
Could we rename to `searchMovies` or something like that?

Reply:
What do you think of `loadMovieOptions`?